package.json

```diff
@@ -14,48 +14,68 @@
     "dist"
   ],
   "scripts": {
-    "build": "npm run clean && webpack",
+    "build": "vite build",
     "clean": "rimraf dist/*",
     "lint": "eslint './src/**/*.{js,jsx,ts,tsx}' --quiet",
-    "start": "webpack watch",
+    "start": "vite build --watch",
     "test": "jest --watchAll"
   },
   "devDependencies": {
     "@babel/cli": "^7.16.7",
     "@babel/core": "^7.16.7",
-    "@babel/plugin-proposal-class-properties": "^7.16.7",
     "@babel/preset-env": "^7.16.7",
     "@babel/preset-react": "^7.16.7",
     "@babel/preset-typescript": "^7.16.7",
-    "@deephaven/eslint-config": "^0.11.0",
-    "@deephaven/prettier-config": "^0.11.0",
-    "@deephaven/stylelint-config": "^0.11.0",
+    "@deephaven/components": "^0.46.0",
+    "@deephaven/dashboard": "^0.46.0",
+    "@deephaven/dashboard-core-plugins": "^0.46.0",
+    "@deephaven/golden-layout": "^0.46.0",
+    "@deephaven/grid": "^0.46.0",
+    "@deephaven/iris-grid": "^0.46.0",
+    "@deephaven/jsapi-types": "^0.46.0",
+    "@deephaven/jsapi-utils": "^0.46.0",
+    "@deephaven/log": "^0.46.0",
+    "@deephaven/prettier-config": "^0.46.0",
+    "@deephaven/stylelint-config": "^0.46.0",
     "@types/jest": "^27.4.0",
     "@types/node": "^17.0.7",
     "@types/react": "^17.0.2",
-    "@typescript-eslint/eslint-plugin": "^4.28.0",
-    "@typescript-eslint/parser": "^5.8.1",
+    "@types/react-dom": "^17.0.2",
+    "@typescript-eslint/eslint-plugin": "^5.46.0",
+    "@typescript-eslint/parser": "^5.46.0",
+    "@vitejs/plugin-react-swc": "^3.0.0",
     "babel-loader": "^8.2.3",
-    "css-loader": "^6.5.1",
-    "eslint": "^7.22.0",
-    "eslint-plugin-es": "^4.1.0",
-    "eslint-plugin-prettier": "^3.3.1",
-    "file-loader": "^6.2.0",
-    "image-webpack-loader": "^8.0.1",
+    "eslint": "^8.29.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^5.0.0",
+    "eslint-plugin-react": "^7.28.0",
     "jest": "^27.4.5",
-    "node-sass": "^7.0.1",
-    "prettier": "^2.2.1",
+    "prettier": "^3.0.0",
+    "process": "^0.11.10",
     "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-hot-loader": "^4.13.0",
+    "react-redux": "^7.2.4",
     "rimraf": "^3.0.2",
-    "sass-loader": "^12.4.0",
-    "style-loader": "^3.3.1",
-    "stylelint": "^13.12.0",
-    "typescript": "^4.5.4",
-    "webpack": "^5.65.0",
-    "webpack-cli": "^4.9.1"
+    "sass": "^1.60.0",
+    "shortid": "^2.2.16",
+    "typescript": "~4.9.4",
+    "vite": "~4.1.4"
   },
   "peerDependencies": {
-    "react": "^17.0.2"
+    "@deephaven/components": "^0.46.0",
+    "@deephaven/dashboard": "^0.46.0",
+    "@deephaven/dashboard-core-plugins": "^0.46.0",
+    "@deephaven/golden-layout": "^0.46.0",
+    "@deephaven/grid": "^0.46.0",
+    "@deephaven/iris-grid": "^0.46.0",
+    "@deephaven/jsapi-types": "^0.46.0",
+    "@deephaven/jsapi-utils": "^0.46.0",
+    "@deephaven/log": "^0.46.0",
+    "@deephaven/redux": "^0.46.0",
```

Comment:
These all aren't necessarily peer deps, but I don't think it matters for this template right now